pkg/client/perseshttp/request.go

```diff
@@ -280,6 +282,7 @@ type RequestError struct {
 	Message    string
 	StatusCode int
 	Err        error
+	OAuthError *api.OAuthError
```

Comment:
I guess you want to include that in the method `Error` no ?

Reply:
I hesitated haha, I see that we're aligned I update to use Err